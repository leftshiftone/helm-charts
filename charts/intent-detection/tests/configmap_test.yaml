suite: test configmap
templates:
  - configmap.yaml
tests:
  - it: default
    set:
      keycloak.enabled: true
    asserts:
      - isKind:
          of: ConfigMap
      - equal:
          path: metadata.name
          value: RELEASE-NAME-intent-detection
      - matchSnapshot:
          path: metadata.labels
      - matchSnapshot:
          path: data
