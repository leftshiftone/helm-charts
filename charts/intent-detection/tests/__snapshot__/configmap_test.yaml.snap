default:
  1: |
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: intent-detection
    app.kubernetes.io/version: 0.0.0
    helm.sh/chart: intent-detection-0.0.0
  2: |
    ENABLE_METRICS: "true"
    HTTP_LOG_LEVEL: WARNING
    LOG_LEVEL: INFO
    MINIO_HOST: semantic-search-minio:9000
    SEMANTIC_SEARCH_INDEXER_URL: http://semantic-search-indexer:8080
    SEMANTIC_SEARCH_SEARCHER_URL: http://semantic-search-searcher:8080
